README.md

```diff
@@ -1,30 +1,40 @@
 # phantom-backend
 Elevate the public transport experience  
 
-# Vision
-Make daily Kigali Commute easy and convenient for the thousands of people who use public transport to move to and from different areas of Kigali.
 
-# What we’re trying to do
-Currently in Rwanda, specifically in Kigali, commuters spend on average 45 minutes at the bus stations or in queues waiting for buses to come, and this number normally goes high in peak hours (7:00 - 8:00, 18:00 - 20:00).
+## Environment variables
 
-Commuters would love a solution to address this problem; they would like to have a way to check the location of buses on their respective routes and plan accordingly so they don’t have to wait for them.
+The app defines three different environment namely.
 
-This is the problem Phantom is here to solve: Create an application that allows simulating bus movements and enabling passengers to track their locations & movements. 
+### Getting started with env 
 
+After cloning or rebasing on main, just create three files named: **.env.development** , **.env.production** and **.env.staging**. 
 
-GETTING STARTED
+Copy all the environment variables shared in **.env.example** file to each of the three files. Assign values to each variable. eg: 
+`PORT=5500` in  `.env.development`
+`PORT=5000` in `.env.production`
+`PORT=6000` in `.env.staging`
```

Comment:
add all this iin .env.example

Reply:
This is a concept if you read clearly you see that you 1 `.env` file is enough and the variable there is `PORT` port is already added in `.env.example`.